src/components/Nav.svelte

```diff
@@ -3,7 +3,120 @@
 </script>
 
 <style>
+  header {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    padding: 12px 0;
+    background-color: #fff;
+    border-bottom: 1px solid hsla(0, 0%, 86%, 0.4);
+    z-index: 999;
+  }
 
+  img {
+    height: 38px;
+    width: 170px;
+  }
+
+  ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    flex-wrap: wrap;
+    padding: 0;
+  }
+
+  li {
+    list-style: none;
+  }
+
+  li a {
+    color: var(--primary);
+    cursor: pointer;
+    font-weight: 500;
+  }
```

Comment:
In global style, make all anchor `cursor: pointer;`

Reply:
Actually, I don't think it's needed? Anchor elements should `cursor: pointer` by default as long as `href` is set, it's just that we don't have a link at the moment.